src/util.test.js

```diff
@@ -1,22 +0,0 @@
-const util = require('../src/util');
```

Comment:
Kompletně chybí testy. Pokud bychom chtěli napsat nějaké unit tests velmi rychle bychom narazili na to, že většina List Elementů je netestovatelná, protože si je nelze separátně importovat (například testování InfoWindow nebo checkBirthDays není možné jinak než testovat to v rámci testování Contacts komponenty. .